biztalk/adapters-and-accelerators/accelerator-swift/server-runtime-security.md

```diff
@@ -50,4 +50,4 @@ manager: "anneta"
 
 - Each digital signature in the stack must be unique. That is, each digital signature must be made by using a unique digital certificate (relative to the other signatures in the same stack). This ensures that the message was not approved by the person who created/repaired the message, and that no person approved the same message as multiple approvers.
 
-  Message Repair and New Submission imposes strict security policies by having checkpoints at each entry point into the message creation, repair, approval, and submission infrastructure. These checkpoints are tightly coupled with core functionality of Message Repair and New Submission and cannot be circumvented. While Message Repair and New Submission is designed to integrate seamlessly with the [!INCLUDE[btsInpathNoVersion](../../includes/btsinpathnoversion-md.md)] form-signing functionality, it is also designed to be secure and robust enough to enforce authenticity and protection of SWIFT messages even if [!INCLUDE[btsInpathNoVersion](../../includes/btsinpathnoversion-md.md)] is not used and messages are submitted directly by end users to Message Repair and New Submission receive endpoints (SharePoint Web folders).
\ No newline at end of file
+  Message Repair and New Submission imposes strict security policies by having checkpoints at each entry point into the message creation, repair, approval, and submission infrastructure. These checkpoints are tightly coupled with core functionality of Message Repair and New Submission and cannot be circumvented. While Message Repair and New Submission is designed to integrate seamlessly with the [!INCLUDE[btsInpathNoVersion](../../includes/btsinpathnoversion-md.md)] form-signing functionality, it is also designed to be secure and robust enough to enforce authenticity and protection of SWIFT messages even if [!INCLUDE[btsInpathNoVersion](../../includes/btsinpathnoversion-md.md)] is not used and messages are submitted directly by end users to Message Repair and New Submission receive endpoints (SharePoint Web folders).
```

Comment:
Is this supposed to be a link to InfoPath?

Reply:
No, this is just adding a new line to the end of the file.